### PR TITLE
nautilus: update to 46.0.

### PR DESCRIPTION
**Pull request by fanyx**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Notes

- [x] https://github.com/void-linux/void-packages/pull/49446

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/nautilus/template [hunk 1]
```diff
@@ -18,10 +18,9 @@ short_desc="GNOME file manager"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later, LGPL-2.1-or-later"
 homepage="https://wiki.gnome.org/Nautilus"
-#changelog="https://gitlab.gnome.org/GNOME/nautilus/-/raw/master/NEWS"
-changelog="https://gitlab.gnome.org/GNOME/nautilus/-/raw/gnome-45/NEWS"
+changelog="https://gitlab.gnome.org/GNOME/nautilus/-/raw/main/NEWS"
```
Comment (oreo639) [inline srcpkgs/nautilus/template:21]: Comment this and add `changelog="https://gitlab.gnome.org/GNOME/nautilus/-/raw/gnome-46/NEWS"` below.

**Comment by fanyx** [comment]
runtime issue:
```
nautilus: symbol lookup error: /usr/lib/libnautilus-extension.so.4: undefined symbol: g_once_init_leave_pointer
```

**Comment by oreo639** [comment]
You probably didn't update libnautilus.

**Comment by fanyx** [comment]
> Try reinstalling libnautilus.

shouldn't that update alongside nautilus itself?
but yeah, you're correct

**Comment by oreo639** [comment]
> actually no, it is installed @`46.0`

Use `xi -f libnautilus`. Also, make sure to clear your `hostdir/binpkgs`.

**Comment by oreo639** [comment]
Oh, sorry, you probably forgot to update glib.

**Comment by fanyx** [comment]
> Oh, sorry, you probably forgot to update glib.

Ohh, right, it's not in the repos yet

**Comment by fanyx** [comment]
yup, works

**Comment by oreo639** [comment]
Thanks for noticing that, should be fixed now: https://github.com/void-linux/void-packages/commit/95f42230ae7b602012ca3eae04e8e624909f5cad

**Review by oreo639 — APPROVED** [review]

**Comment by fanyx** [comment]
> Thanks for noticing that, should be fixed now: [95f4223](https://github.com/void-linux/void-packages/commit/95f42230ae7b602012ca3eae04e8e624909f5cad)

yeah i was wondering about that when building glib
thought it was fine though as when a newer version is available that one would be pulled/installed
guess it wasn't :D

**Comment by oreo639** [comment]
Yeah, it only automatically upgrades dependencies if they are necessary to match `glib>=minver`.

**Pull request merged**